.github/workflows/docker-release.yaml

```diff
@@ -10,9 +10,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        cassandra-version: [6.8.25]
+        dse-version: [6.8.25, 6.8.26]
```

Comment:
I'm not sure if we want/need to go back to 6.8.25 or not. The list here should only contain DSE versions we will support going froward. Also, since DSE images built by this action are only for testing, maybe we don't even need to maintain old versions here?

Reply:
not sure either... We may want to leave it like this for now though.

Reply:
No real harm in doing so.